content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -146,7 +146,7 @@ All modifications to a cron job, especially its `.spec`, are applied only to the
 The `.spec.schedule` is a required field of the `.spec`.
 It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`, as schedule time of its jobs to be created and executed.
 
-The format also includes extended `vixie cron` step values. As explained in the
+The format also includes extended “Vixie cron” step values. As explained in the
```

Comment:
Hi @sftim .
One nit: Should the text use this double quote symbol, `"Vixie cron"`?
There was a previous PR that cleaned up quotes:
https://github.com/kubernetes/website/pull/22981

Reply:
I quite like “proper” quotes, but I can fix it easily enough.